src/oatpp-swagger/oas3/Generator.cpp

```diff
@@ -102,6 +102,10 @@ oatpp::Object<Schema> Generator::generateSchemaForSimpleType(const Type* type, T
     if(!property->info.description.empty()) {
       result->description = property->info.description.c_str();
     }
+    // if(!property->info.defaultValue.empty()) {
+    //   result->defaultValue = property->info.defaultValue.c_str();
+    // }
+    // Should use defaultValue but don't know how
```

Comment:
Let's try to make `result->defaultValue` of type `Any` and then just assign it directly to it.

```cpp
result->defaultValue = property->info.defaultValue; // should work
```

Reply:
Will try these as soon as possible
Thanks!